include/glow/Partitioner/Partitioner.h

```diff
@@ -161,6 +164,12 @@ class Partitioner final : public PartitionerBase {
   /// for function optimization. \returns the partition result or an error.
   llvm::Expected<DAGListTy> heterogeneousPartition(CompilationContext &cctx);
 
+  /// This partition approach is an experimental one. It tries to balance the
+  /// workloads of each accelerator/device in addition to respecting memory
+  /// constraints.
+  llvm::Expected<DAGListTy> loadBalancedPartition(CompilationContext &cctx,
+                                                  size_t numDevices = 0);
```

Comment:
can you document numDevices. It seems that it gets overwritten under certain conditions inside the LBPartition.